src/extra/AutoUi/Collection/index.tsx

```diff
@@ -68,17 +76,30 @@ const HeaderGrid = styled(Flex)`
 	}
 `;
 
-const getSelectedItems = <T extends AutoUIBaseResource<T>>(
-	newItems: T[],
-	selectedItems: T[],
-) => {
-	if (!size(selectedItems)) {
-		return selectedItems;
+const Focus = styled(Box)`
+	flex-basis: 500px;
+	background-color: white;
+	border: solid 1px ${(props) => props.theme.colors.quartenary.dark};
+	max-height: 200px;
+	position: absolute;
+	width: 100%;
+	z-index: 1;
+	border-radius: 0 0 5px 5px;
```

Comment:
0 0 props.theme.global.drop.border.radius props.theme.global.drop.border.radius long but at least we will use the same border radius for every drop

Reply:
Didn't know we had this, thanks!!